carbon/components/MobileTabSelector/index.tsx

```diff
@@ -11,15 +11,15 @@ export const MobileTabSelector = <T extends string>(props: {
   value: string;
   options: Array<TypedOption<T>>;
   onSelectionChanged: (value: T) => void;
-  className: string;
+  className?: string;
 }) => {
   const handleChange = (event: SelectChangeEvent) => {
     props.onSelectionChanged(event.target.value as T);
   };
 
   return (
     <Select
-      className={`${styles.select} ${props.className}`}
+      className={`${styles.select} ${props.className ?? ""}`}
```

Comment:
I do not thing you need the  `?? ""` part

Reply:
I forgot to consider this, I’ll create a separate PR. I’m sorry!